src/Network/Network/Generated/Models/PSVirtualRouterPeer.cs

```diff
@@ -0,0 +1,43 @@
+﻿// <auto-generated>
```

Comment:
Since it's a manually written model, could you please remove parts about generation from the header?

Reply:
Done